one_fm/templates/emails/leave_application_details_for_employee.html

```diff
@@ -0,0 +1,146 @@
+
+                    <div class="section">
+                        <strong>Leave Duration Request:</strong>
+                        <ul>
+                            <li><strong>Start Date:</strong> {{ from_date }}</li>
+                            <li><strong>End Date:</strong> {{ to_date }}</li>
+                            <li><strong>Total Days:</strong> {{ total_leave_days }} days</li>
+                        </ul>
+                    </div>
+
+                    <div class="section">
+                        <div><strong>Date of Application: </strong>{{ date_of_application }}</div>
+                        <div><strong>Leave Approver: </strong>{{ leave_approver }}</div>
+                        <div><strong>Status: </strong>{{ status }}</div>
+                    </div>
+
+                    <p>Please note that your leave request is currently under review. You will be notified once your leave has been approved or if any further action is required.</p>
+                    <p>If you have any questions or need to make adjustments to your request, please contact your line manager.</p>
+                </div>
+            </td>
+            <td>
+                <div class="container arabic">
+                    <p>عزيزي/عزيزتي {{ employee_name_arabic }}،</p>
+
+                    <div class="section">
+                        <p>لقد قدمت طلب إجازة، والتفاصيل أدناه:</p>
+                        <div><strong>اسم الموظف:</strong> {{ employee_name_arabic }}</div>
+                        <div><strong>رقم الموظف:</strong> {{ employee_id }}</div>
+                        <div><strong>نوع الإجازة:</strong> {{ _(leave_type_eng, "ar") }}</div>
+                    </div>
+
+                    <div class="section">
+                        <strong>مدة الإجازة الأصلية:</strong>
+                        <ul>
+                            <li><strong>تاريخ البدء:</strong> {{ from_date }}</li>
+                            <li><strong>تاريخ الانتهاء:</strong> {{ to_date }}</li>
+                            <li><strong>إجمالي الأيام:</strong> {{ total_leave_days }} أيام</li>
+                        </ul>
+                    </div>
+
+                    <div class="section">
+                        <div><strong>تاريخ التقديم:</strong> {{ date_of_application }}</div>
+                        <div><strong>الموافق:</strong> {{ leave_approver }}</div>
+                        <div><strong>الحالة:</strong> {{ status }}</div>
+                    </div>
+
+                    <p>يرجى ملاحظة أن طلب إجازتك قيد المراجعة حاليًا. ستتلقى إشعارًا بمجرد الموافقة على إجازتك أو إذا كانت هناك حاجة لأي إجراء إضافي.</p>
+                    <p>إذا كان لديك أي أسئلة أو تحتاج إلى إجراء تعديلات على طلبك، يرجى الاتصال بالمدير المباشر الخاص بك.</p>
+                </div>
+            </td>
+        </tr>
+        <tr>
+            <th>Document Link</th>
+            <td><a href="{{ base_url }}/app/leave-application/{{ doc_name }}" target="_blank">Click Here</a></td>
```

Comment:
If you use the get_url_to_form, you would not need to hard code, "app/leave-application", you could just pass it in directly